src/constant.ts

```diff
@@ -1 +1,48 @@
 export const COSMOSDB_DOC_URL = "https://docs.microsoft.com/azure/cosmos-db/introduction";
+
+export const Telemetry = {
+  // all events will be prefixed with this event name
+  extensionId: "azure-cosmosdb-ads-extension",
+  // extension version will be reported as a property with each event
+  extensionVersion: "0.1.1",
```

Comment:
It'd be better to read in the package.json and pull the version from there, that way you're sure not to miss updating this value each time you vBump. See https://github.com/microsoft/azuredatastudio/blob/main/extensions/admin-tool-ext-win/src/telemetry.ts#L14 for an example

(same with package ID but that's less likely to change so up to you)

Reply:
Good point.